src/Application.php

```diff
@@ -340,6 +343,13 @@ protected function discoverCommandsFromConfiguration()
                 $commandList[$value] = $classname;
             }
         }
+
+        if ($commandList) {
+            $message = dt('Discovering commands by configuration is deprecated in Drush 11.0.9 and is scheduled for removal in a future major version. The following command classes should be converted to PSR4 discovery (see docs/commands.md): !classes', ['!classes' => implode(', ', $commandList)]);
```

Comment:
I would give a full URL to the docs. A relative path is not giving the user much info, as our code is bured in vendor.